.Documentation/articles/innoactive-creator/default-conditions.md

```diff
@@ -23,84 +25,135 @@ See [this article](step-inspector.md) to learn Step Inspector controls.
 
 ### Description
 
-The `Grab Object` condition is fulfilled once the specified `Training Scene Object` is grabbed while this condition is active. It is fulfilled immediately if the condition becomes active while the `Training Scene Object` is already being held.
+This condition is fulfilled when the trainee grabs the `Object`. 
+The condition is also fulfilled if the trainee already grabs the `Object` before the step is activated, so, if a trainee is already holding 
+the specified object in hand while this condition is active, it is fulfilled immediately.
 
-### Configuration
+### Interaction in VR
+
+To grab the object in VR, by default, the trainee has to move one of their hands/controllers into the object (it’s collider) and press the trigger button. The button to trigger can be changed by the template developer as described here (TO BE DONE).
+
+
+### Interaction in Desktop Mode
+
+To grab the object in Desktop Mode, the trainee has to move the mouse on the object (so that the cast ray hit’s the objects collider) and press the left mouse button. The button to trigger can be changed by the template developer as described here (TO BE DONE).
+
+### Interaction Highlights
+Before the controller button or, respectively, mouse button is pressed, by default, the object is already highlighted to indicate that the trainee can now grab this object. This “on hover” effect can be changed by the template developer here (TO BE DONE). The training designer can also change this in the interactable highlighter property of the given object as described here. You can easily enable/disable this hover effect, or change the highlight color and transparency (TO BE DONE).
 
-- #### Grabbable object
+### Application Example
 
-    This field contains the `Training Scene Object` with a `Grabbable Property` that is required to be grabbed.
-    Besides, the object must have a collider component in order to be grabbed.
+- Trainees need to grab objects to position them somewhere.
+- Trainees need to grab a tool to use it.
+- Trainees need to grab a component part to assemble it to another component part.
 
-    [![Grabbable Properties](../images/conditions/grabbable_properties.png "")](../images/conditions/grabbable_properties.png)
 
+### Configuration
+
+- #### Object
+
+    The `Training Scene Object` to grab. The object needs to have the `Grabbable Property` and a collider component configured. The collider defines the area where the trainee can grab this object.
+
+### Location of this Condition (for Developers)
+This behavior is part of the <a href="https://github.com/Innoactive/Basic-Interaction-Component" target="_blank">Basic-Interaction</a> component. The file is located <a href="https://github.com/Innoactive/Basic-Interaction-Component/blob/develop/Runtime/Conditions/GrabbedCondition.cs" target="_blank">here</a>
```

Comment:
this is not how you do links in md 🤔 

`[Basic-Interaction](https://github.com/Innoactive/Basic-Interaction-Component)`

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links